crates/pop-contracts/src/errors.rs

```diff
@@ -7,55 +7,87 @@ use thiserror::Error;
 #[derive(Error, Debug)]
 #[allow(clippy::enum_variant_names)]
 pub enum Error {
+	/// An error occurred.
```

Comment:
Need the comments because we are exposing this for the integration tests (file below)